### ISSUE TITLE
Question on KITTI training

### ISSUE THREAD

**Issue by AimAlex** [ucbdrive/hd3#14]
Hi. In the stereo matching task, you trained the model on KITTI for 2000 epochs. But do you have a validation dataset? How could you know which epoch is the best model? Are you using the latest model to test?

I got similar result on the training set, and used the latest model to run the test. However the result is not as good as yours on the test dataset.

Thank you.

**Comment by yzcjtr** [comment]
There's no validation set on KITTI. We just use the model of the latest epoch for testing. How is your result on the test set of KITTI? Did you use the context module?

**Comment by AimAlex** [comment]
![屏幕快照 2019-08-01 上午11 12 21](https://user-images.githubusercontent.com/26958630/62316859-51d66f80-b44d-11e9-98b1-1642f0e305b7.png)

This is my result and I used the context module.

**Comment by yzcjtr** [comment]
Looks good. I think the result is plausible, considering different random seeds and the scarcity of training data on KITTI. Besides, numbers in stereo matching on the KITTI benchmark is saturated, making subtle differences not that important.
